.gitignore

```diff
@@ -76,3 +76,6 @@ yarn-error.log
 ###< symfony/phpunit-bridge ###
 
 local-php-security-checker
+*.crt
+*.key
```

Comment:
I can imagine (test) scenarios where we want to keep a cert or even a private key in vcs. Having this rigid rule in place will make it harder to add them. For now, feel free to leave this as is. But a good rule of thumb is to be specific in .gitignore files. Not being to loose in excluding things.

Reply:
It has been changed to specifically ignore the tiqr.stepup.example.com key and crt